Tensile/KernelWriterAssembly.py

```diff
@@ -6819,8 +7105,13 @@ def calculateLdsWriteOffset(self, perp, para, sPerp, sPara, kernel, tP, localWri
       lspaOffset *= (kernel[tP["mt"]] + kernel["LdsPad%s"%tc])
       lspaOffset += rem * ldl + perp_rem
     else:
-      lscaOffset *= (kernel[tP["mt"]] + kernel["LdsPad%s"%tc])
-      lscaOffset += rem
+      if not kernel["TransposeLDS"]:
+        lscaOffset *= (kernel[tP["mt"]] + kernel["LdsPad%s"%tc])
+        lscaOffset += rem
+      else:
+ 	#FIXME re-verify lscaOffset=0
```

Comment:
Is this done?

Reply:
may need to revisit with extensive testing..